bootstrap-app/src/cmr/bootstrap/api/routes.clj

```diff
@@ -133,8 +133,7 @@
               (= keyword-cache-name provider-cache/cache-key)
               (provider-cache/refresh-provider-cache request-context)
 
-              (or (= keyword-cache-name coll-for-gran-acls-caches/coll-by-concept-id-cache-key)
-                  (= keyword-cache-name coll-for-gran-acls-caches/coll-by-provider-id-and-entry-title-cache-key))
+              (= keyword-cache-name coll-for-gran-acls-caches/coll-by-concept-id-cache-key)
```

Comment:
What happens if someone requests the entry-title-cache-key refresh?

Reply:
I've removed the cache entirely, so it should not be called anywhere

Reply:
that path won't exist anymore and neither will that cache